src/briefcase/app.py

```diff
@@ -406,7 +406,7 @@ def install_launch_scripts(self):
                 from pkg_resources import load_entry_point
 
                 if __name__ == '__main__':
-                    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
+                    sys.argv[0] = re.sub(r'(-script\\.pyw?|\\.exe)?$', '', sys.argv[0])
```

Comment:
This string is already a "raw" string with the `r''` added. Do we need to double escape the two regex special characters for this line?

Reply:
Yes, because the entire string is inside another string itself. It starts at line 393, this is not really readable just within this snippet :)

Reply:
@BoboTiG Thanks for the explanation, I missed that. I needed to look a bit further up and down :smile: